### ISSUE TITLE
New conformance requirement unreasonable

### ISSUE THREAD

**Issue by Wildebrew** [w3c/wcag21#722]
There is a new note in the conformance section of WCAG 2.1 
"
New A full page includes each variation of the page that is automatically generated by the page for various screen sizes. Each of these variations needs to conform (or needs to have a conforming alternate version) in order for the entire page to conform."

I believe this requirement is too strict and places and undue burden on website owners.
We all agree that accessibility verification testing of a webpage requires between 1 and 4 hours of work by an expert tester. If a website is responsive and has 5 breakpoints that adds up to 20 hours of additional work (probably more).
Testing responsive break points of webpages also bring into play additional nuances related to touch screen accessibility and so on.
I believe requiring all breakpoints of the page to conform is unreasonable, (businesses will have a much stronger case for accessibility being too costly when the work multiplies).
I also believe this will dilue accessibility efforts and businesses will seak to achieve 90% conformance on all breakpoints, whereas it is better for the user if a version of the page is 100% conforming at one breakpoint.

I believe, instead, that at least 2 versions of a webpage, the desktop and one responsive breakpoint (I'd recommend the smallest one) are required to conform.
Either that, or additional conformance can be claimed for each responsive version of the page.
I am worried, given my experience working with various big and small businesses, that this requirement will do more harm than good

**Comment by awkawk** [comment]
Thank you for the comment. The deadline for comments has passed, but the Working Group will make efforts to consider your comment and any suggestions for improvements.

**Comment by mraccess77** [comment]
This is a personal unofficial explanation related to user impact for people with low vision.  When users with low vision zoom the page on the their desktop or laptop or use low resolution like myself we end up in with different breakpoints along the way to 200% zoom.   If those breakpoints are not accessible and there is no way to get to a version of the page that can be zoomed without putting me into an in accessible page then it prevents me from using zoom on those web pages.  Users should be able to choose their zoom level from 100 to 200 percent and still have access to an accessible page.

**Comment by Wildebrew** [comment]
Andrew
I'm sorry, I thought the deadline was "by January 12". Now I see it is
"before" January 12. My bad.
I had two more issues I wanted to file. I suppose I'll do that anyway,
and the group can take them into consideration if possible.
Congrats on taking WCAG forward. I know this is an enormous effort and
I am overall very happy with what this looks like.




On 1/12/18, Andrew Kirkpatrick <notifications@github.com> wrote:
> Thank you for the comment. The deadline for comments has passed, but the
> Working Group will make efforts to consider your comment and any suggestions
> for improvements.
>
> --
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/w3c/wcag21/issues/722#issuecomment-357253302


-- 
Work hard. Have fun. Make history.

**Comment by awkawk** [comment]
(Official WG Response)
Thank you for your comment. The Working Group came to a different conclusion, including others from your organization. We hope to gain further data to support this change in the implementation testing during CR.
